scripts/components/pixel-canvas.js

```diff
@@ -35,6 +35,56 @@ class PixelCanvasProto extends HTMLElement {
   }
 
   /**
+    Fetch the current canvas state and turn it into a
+    JSON string representing the canvas.
+
+    First we turn the data into a 2D array where each
+    entry is the color hash of the pixels current
+    background color. Then we JSON.stringify that array.
+
+    @return {string} - a string produced by calling JSON.strigify on
+      a 2D Array containing color hex values, such as '#ffffff' for white.
+  */
+  serializeGrid() {
```

Comment:
Great question, in the ES6 Class Syntax, adding the keyword `function` here actually results in an error -- which is a bit of a departure from JavaScript's old standard. 

Checkout the specification at MDN for more information on why this is the pattern for the `class` syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes